standard/interfaces.md

```diff
@@ -1066,7 +1066,7 @@ When a generic method implicitly implements an interface method, the constraints
 
 ### 19.6.5 Interface mapping
 
-A class or struct shall provide implementations of all abstract members of the interfaces that are listed in the base class list of the class or struct. The process of locating implementations of interface members in an implementing class or struct is known as ***interface mapping***.
+A class or struct shall provide implementations for all abstract members of the interfaces that are listed in the base class list of the class or struct which are not implemented in any of those listed interfaces. The process of locating implementations of interface members in an implementing class or struct is known as ***interface mapping***.
```

Comment:
Fix typo/language. Limit the requirement now that abstract members can be implemented in interfaces.